### PR TITLE
Design changes and fixes

### PR DESCRIPTION
**Pull request by rakshakhegde**
Confirmed with Pragati that this feature has to be changed to accept 4 digits. So I'll skip this commit and make this change in another PR.
~https://www.pivotaltracker.com/story/show/171296031~
https://www.pivotaltracker.com/story/show/171854372
https://www.pivotaltracker.com/story/show/170942753

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -519,7 +519,7 @@
   <string name="bloodsugarentry_error_higher_limit">Value cannot be more than 1000 mg/dl</string>
   <string name="bloodsugarentry_error_higher_limit_hba1c">Value cannot be more than 25%</string>
   <string name="bloodsugarentry_error_empty">Value cannot be empty</string>
-  <string name="bloodsugarentry_error_date_invalid_pattern">Enter a valid date in DD/MM/YY</string>
+  <string name="bloodsugarentry_error_date_invalid_pattern">Enter a valid date in DD/MM/YYYY</string>
```
Comment (sanchita) [inline app/src/main/res/values/strings.xml]: The pivotal story says that :
"Backdating a BP error message says the date must be DD/MM/YYYY, but it actually requires a 2 digit date".
It is a 2 digit date. Why are we changing it to 4 digits?

File: app/src/main/java/org/simple/clinic/newentry/PatientEntryScreen.kt [hunk 2]
```diff
@@ -255,6 +260,14 @@ class PatientEntryScreen(context: Context, attrs: AttributeSet) : RelativeLayout
     consentTextView.text = consentText
   }
 
+  private fun setConsentLabelText() {
+    val consentLabelTextResId: Int = when (country.isoCountryCode) {
+      Country.BANGLADESH -> R.string.patiententry_consent_sms_reminders
+      else -> R.string.patiententry_consent_whatsapp_sms_reminders
+    }
+    consentLabel.setText(consentLabelTextResId)
```
Comment (sanchita) [inline app/src/main/java/org/simple/clinic/newentry/PatientEntryScreen.kt]: Can we refactor this to remove the type `Int` and change `setText` to `text` in order to make it more Kotlin idiomatic?
```
    val consentLabelTextResId = when (country.isoCountryCode) {
      Country.BANGLADESH -> R.string.patiententry_consent_sms_reminders
      else -> R.string.patiententry_consent_whatsapp_sms_reminders
    }
    consentLabel.text = resources.getString(consentLabelTextResId)
```

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (rakshakhegde) [reply app/src/main/res/values/strings.xml]: The pivotal story says that right now the app has only 2 digits but it must actually be YYYY.

File: app/src/main/java/org/simple/clinic/newentry/PatientEntryScreen.kt (hunk 2 above)
Comment (rakshakhegde) [reply app/src/main/java/org/simple/clinic/newentry/PatientEntryScreen.kt]: This doesn't really change much. I prefer to keep it as it is.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (sanchita) [reply app/src/main/res/values/strings.xml]: No. We enter 2 digits. Can you please check the app?

File: app/src/main/java/org/simple/clinic/newentry/PatientEntryScreen.kt (hunk 2 above)
Comment (sanchita) [reply app/src/main/java/org/simple/clinic/newentry/PatientEntryScreen.kt]: It does not make sense to have the type specified as it is derived already.
Can you please make that change atleast?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (rakshakhegde) [reply app/src/main/res/values/strings.xml]: You are right. Confirmed with Pragati that this feature has to be changed to accept 4 digits. So I'll skip this commit and make this change in another PR.

**Pull request merged**